### PR TITLE
[MOBILE-178] Add missing doc and cleanup xamarin component cruft

### PR DESCRIPTION
**Pull request by BrianBatchelder**
The Xamarin Component Store no longer exists. This cleans up some stuff we used to build our Xamarin component.

### CODE REVIEW

File: documentation/migration-guide-fcm.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+FCM migration is optional, but recommended as Google deprecated GCM and is planning on dropping
+GCM client APIs in April of 2019. For more info, please read the [GCM F.A.Q](https://developers.google.com/cloud-messaging/faq).
+
+Urban Airship supports the FCM client library starting in SDK 9.1.0. It is available in a new package
+`urbanairship.android.fcm`. The FCM package is not compatible with the deprecated package `UrbanAirship`.
+
+Migration Steps:
+
+- [Add Firebase to your app](https://firebase.google.com/docs/android/setup#add_firebase_to_your_app).
```
Comment (rlepinski) [inline documentation/migration-guide-fcm.md]: Lets avoid their guide

Comment (BrianBatchelder) [reply documentation/migration-guide-fcm.md]: @rlepinski: I got that from our Android SDK FCM Migration Guide. Should I just point there instead?

Comment (rlepinski) [reply documentation/migration-guide-fcm.md]: My bad, i though it was the bad xamarin guide

**Comment by marc-scig** [comment]
🏋️

**Comment by rlepinski** [comment]
👍

**Pull request merged**